invisible_cities/detsim/sensor_utils.py

```diff
@@ -34,7 +34,7 @@ def create_timestamp(event_number: Union[int, float],
         period = 1 / rate
         timestamp = abs(event_number * period) + np.random.uniform(0, period)
     except ZeroDivisionError:
-        ## Maybe a warning here
+        print(f"WARNING: rate = {rate}")
```

Comment:
Now that I see it in place I think that putting a warning here (even a proper one) might be unnecessarily verbose. Maybe it's a warning that should go in buffy itself or in the component so it's only sent once per job for the records.